Makefile

```diff
@@ -1,74 +1,79 @@
+platform != uname -s
+undefine != ./detect-os-symbol-list-file.sh undefine
```

Comment:
Seems okay, but maybe we can just do undefine `undefine-$(shell uname).lst`? I dunno if that is portable though!

Reply:
AFAIK it should be portable between GNU and BSD make , but then on systems for which you don't have an explicit list file for the build will error out with file not found. The shell scripts provides a default if the system is not known to it.